src/06-products/products.apib

```diff
@@ -24,3 +24,5 @@ Get a list of products.
                         + name: `cookies` (string)
                         + description: `dark chocolate` (string)
                 + external_id: `123` (string)
+                + purchase_price (Money)
```

Comment:
I believe you forgot to build the documentation it should also add it to the `https://github.com/teamleadercrm/api/blob/master/apiary.apib`

Reply:
Its built now.